x-pack/plugins/security_solution/server/routes/index.ts

```diff
@@ -138,13 +139,20 @@ export const initRoutes = (
   // Dashboards
   registerDashboardsRoutes(router, logger);
   registerTagsRoutes(router, logger);
-  const { previewTelemetryUrlEnabled } = config.experimentalFeatures;
+
+  const { previewTelemetryUrlEnabled, siemMigrationsEnabled } = config.experimentalFeatures;
+
   if (previewTelemetryUrlEnabled) {
     // telemetry preview endpoint for e2e integration tests only at the moment.
     telemetryDetectionRulesPreviewRoute(router, logger, previewTelemetryReceiver, telemetrySender);
   }
 
   registerEntityAnalyticsRoutes({ router, config, getStartServices, logger });
+
+  if (siemMigrationsEnabled) {
+    registerSiemMigrationsRoutes(router, logger);
+  }
```

Comment:
Nit: can we register conditionally inside the `registerSiemMigrationsRoutes` function itself? That way next time we need to remove the feature flag check, it won't trigger additional CODEOWNERS reviews.